composer/metrics/nlp.py

```diff
@@ -52,7 +52,7 @@ def compute(self):
         return self.correct.float() / self.total
 
 
-class CrossEntropyLoss(Metric):
+class LanguageCrossEntropyLoss(Metric):
```

Comment:
Similar to the other, if this is only meant to be used as a metric for evaluation, it might make more sense to call it `LanguageCrossEntropy`. If it gets used as a loss during training I'd expect it to be in `loss.py` or the nlp equivalent.